projects/core/src/i18n/translate.pipe.spec.ts

```diff
@@ -1,20 +1,31 @@
-import { ChangeDetectorRef } from '@angular/core';
+import { ChangeDetectorRef, Injectable } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
 import { EMPTY, of } from 'rxjs';
+import { MockTranslationService } from './testing/mock-translation.service';
 import { TranslatePipe } from './translate.pipe';
 import { TranslationService } from './translation.service';
-import createSpy = jasmine.createSpy;
+
+@Injectable()
+class MockChangeDetectorRef implements Partial<ChangeDetectorRef> {
+  markForCheck() {}
+}
```

Comment:
[MEDIUM]

Are changes in this file _really_ related to this PR?

Reply:
yes, this change is related to this PR. Previously TranslatePipe wasn't tested as an injectable and a new instance was created with the `new` operator. Such a way was causing an issue with the `inject` function used to inject `LoggerService`. When I provided `TestBed` to this test, I had to create a few mocks.